packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -126,15 +126,15 @@ Okta defines various reserved claims that can't be overridden. When you add a cu
 
 | Claim name     | Token type        |
 |----------------|-------------------|
-| acr            | access token      |
-| amr            | access token      |
+| authorization_details | access token      |
 | as_uri         | access token      |
-| cid            | access token      |
-| groups         | access token      |
+| cnf (if [DPoP](/docs/guides/dpop/main) enabled) | access token      |
```

Comment:
I will leave this up to a Doc reviewer to decide if this is how we want to doc it. Just pointing out that this claim is already present for ID token as well. https://github.com/okta/okta-developer-docs/pull/4250/files#diff-f2d1b4f12feefd6381dc18b2d147e1f3a34a2817818a7f0e0ee4751c5706f363L150

Reply:
Yes, I left `cnf` in the "ID token" section that follows. But I could also add to the "both" portion of the table with the DPoP caveat.